packages/ember-glimmer/lib/syntax/curly-component.js

```diff
@@ -259,6 +262,10 @@ class CurlyComponentManager {
   }
 
   didCreateElement({ component, classRef, environment }, element, operations) {
+    if (environment.isInteractive) {
+      component.trigger('willRender');
+    }
```

Comment:
I think the difference is probably not observable. All that would have happened between the end of create and here is that we would have compiled the layout, entered it, created an empty element for in the VM (but haven't appended it into DOM). So it should be okay to align this particular book between tagless and regular components.